scripts/sentiment_analysis/sentiment_analysis_cnn.py

```diff
@@ -3,11 +3,12 @@
 ===============================================
 This example shows how to use convolutional neural networks (textCNN)
 for sentiment analysis on various datasets.
-
 Kim, Y. (2014). Convolutional neural networks for sentence classification.
 arXiv preprint arXiv:1408.5882.
 """
 
+# coding: utf-8
```

Comment:
This is only needed in Python 2. But we don't support Python 2, so you can remove it.

Reply:
Ok, done.